### DISCUSSION TITLE
old pc crashed, cannot export, but I have files under firefox profile folder

### DISCUSSION THREAD

**Discussion by tustinn** [openstyles/stylus#1340] [Q&A]
My old PC crashed and I cannot export any files, but was able to save the files under the firefox profiles folder.  I'm having a hard time restoring it to new firefox profile, if I copy/paste and go to "Manage," the extension give me a database error.  Any suggestions on how to transfer the database files to new PC?  Need to access the CSS I spent hours on.  Thanks.

**Comment by tophf** [comment]
Try #1094.

**Reply by tustinn** [reply]
> 
> 
> Try #1094.
Will check, thanks!
